sklearn/metrics/classification.py

```diff
@@ -1543,7 +1543,8 @@ def hamming_loss(y_true, y_pred, classes=None, sample_weight=None):
         raise ValueError("{0} is not supported".format(y_type))
 
 
-def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None):
+def log_loss(y_true, y_pred, labels=None, eps=1e-15, normalize=True,
```

Comment:
if log_loss has already been released then labels param should be move to the end as it's an API change.